src/main/java/seedu/duke/Duke.java

```diff
@@ -7,23 +7,52 @@
 import seedu.duke.data.ShoppingList;
 import seedu.duke.parser.Parser;
 
+import java.io.File;
+import java.io.IOException;
 import java.util.Scanner;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
 
 public class Duke {
 
-    /**
-      * Main entry-point for the java.duke.Duke application.
-      *
-     */
     public static Budget myBudget = new Budget();
     private static ShoppingList items = new ShoppingList();
     private static Scanner in = new Scanner(System.in);
+    private static final Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+    public static final String DEFAULT_FILEPATH = "Logger.log";
+    public static final String LOG_FILE_ERROR = "Logging to file unsuccessful";
```

Comment:
Maybe all these here can be private instead?

Reply:
Alright will make those changes!